css/style.css

```diff
@@ -0,0 +1,42 @@
+header, ul{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+}
+#header-img{
+    width: 400px;
+}
+.nav-link {
+    text-decoration: none;
+}
```

Comment:
Your code could be more readable if you avoid making whitespace before the property text-decoration 👍